File: pl-domain/exam-service/src/main/java/com/polylab/exams/domain/service/ExamServiceImpl.java
package com.polylab.exams.domain.service;

import com.jxareas.jxcore.domain.service.PersistentService;
import com.polylab.exams.persistence.model.Exam;
import com.polylab.exams.persistence.repository.ExamRepository;
import org.springframework.stereotype.Service;

@Service
public class ExamServiceImpl extends PersistentService<Exam, Integer> implements ExamService {
    public ExamServiceImpl(ExamRepository repository) {
        super(repository);
    }
}

File: pl-domain/exam-service/src/main/java/com/polylab/exams/domain/service/ExamServiceImpl.kt
package com.polylab.exams.domain.service

import com.jxareas.jxelerate.domain.service.PersistentService
import com.polylab.exams.persistence.model.Exam
import com.polylab.exams.persistence.repository.ExamRepository
import org.springframework.stereotype.Service

@Service
class ExamServiceImpl(repository: ExamRepository) :
    PersistentService<Exam, Int>(repository), ExamService
